src/pstore/pstore.c

```diff
@@ -227,7 +227,7 @@ static int process_dmesg_files(PStoreList *list) {
                 if (!startswith(pe->dirent.d_name, "dmesg-"))
                         continue;
 
-                if ((p = startswith(pe->dirent.d_name, "dmesg-efi-"))) {
+                if ((p = STARTSWITH_SET(pe->dirent.d_name, "dmesg-efi-", "dmesg-efi_pstore-"))) {
```

Comment:
can you add a comment right here that explains that one is the old name, and the other the name, and mention the kernel number was changed in. And a brief passive-aggressive comment about kernel people breaking API would get my blessing too on this ;-)

Reply:
> And a brief passive-aggressive comment about kernel people breaking API

lol. I didn't fully understand the kernel change, though